### DISCUSSION TITLE
Foooocus for Cascade / SD3

### DISCUSSION THREAD

**Discussion by JF1805** [lllyasviel/Fooocus#2366] [Q&A] [answered]
Hi team,

are there any plans to keep Fooocus up to speed by adapting it to Stable Cascade or SD3 when these are released?
We are seeing so many incredible new approaches for low power hardware! :-)
Very exciting

Thanks and best regards

**Comment by blckbx** [comment, marked as answer]
Afaik for Cascade support Fooocus would require a redesign of its internal workflow to process this staged approach. 

For low power hardware there's now SDXL Lightning which requires very few steps to achieve quite good results.
There are quite a few options available now. Juggernaut XL, DreamShaper XL, RealVisXL also provide Lightning versions:
https://civitai.com/models/139562/realvisxl-v40?modelVersionId=361593
https://civitai.com/models/112902?modelVersionId=354657
https://civitai.com/models/133005/juggernaut-xl?modelVersionId=357609
Download the checkpoint, place it in `/models/checkpoints` refresh Fooocus to load them and configure steps and some debug settings as described by your favorite model.

For SD3 there's no technical report out yet. We will see when we get there.
